src/Compilers/CSharp/Portable/Lowering/DiagnosticsPass_Warnings.cs

```diff
@@ -212,7 +212,10 @@ private static bool IsSameLocalOrField(BoundExpression expr1, BoundExpression ex
 
         private static bool IsComCallWithRefOmitted(MethodSymbol method, ImmutableArray<BoundExpression> arguments, ImmutableArray<RefKind> argumentRefKindsOpt)
         {
-            if ((object)method.ContainingType == null || !method.ContainingType.IsComImport) return false;
+            if (method.ParameterCount != arguments.Length ||
+                (object)method.ContainingType == null || 
```

Comment:
The check existed before the change. Since we get here with error method symbols, I would not be surprised if they do not have containing types.